assets/docs/tulasiram58827/keras-ocr/float16/1.md

```diff
@@ -1,11 +1,10 @@
 # Lite tulasiram58827/keras-ocr/float16/1
-
 TF Lite quantized version of the CRAFT model for text detection from natural scenes and documents.
 
 <!-- parent-model: tulasiram58827/keras-ocr/1 -->
-<!-- asset-path: https://github.com/tulasiram58827/ocr_tflite/releases/download/v0.1/keras_ocr_float16.tar.xz -->
+<!-- asset-path: https://github.com/tulasiram58827/ocr_tflite/releases/download/v0.2/keras_ocr_float16_ctc.tar.xz -->
```

Comment:
My suggestion for this PR is equivalent to the one in https://github.com/tensorflow/tfhub.dev/pull/31/files#r553198934